### ISSUE TITLE
Correct way to use --latest-stamps

### ISSUE THREAD

**Issue by schleeb** [instaloader/instaloader#1281]
I was really excited when I read about the `--latest-stamps` feature, since I often clean out items I'm not interested in keeping, but that can be annoying if it happened to be the most recent item that was posted, however, when I went to use it in place of `--fast-update`, I was greeted with:  `No targets were specified, thus nothing has been downloaded.`

It did seem to work when I paired it w/ `--fast-update`, but the documentation gave the impression it worked similarly to, but not in addition to it.

Can someone please provide the correct method of use, or explain what I'm doing wrong?

Thank you, and great work on this!

**Comment by ekalin** [comment]
You didn't post the command line, so I have to guess. I'd say you're calling
```
instaloader.py --latest-stamps selenagomez
```

`--latest-stamps` takes an optional argument, so it's considering `selenagomez` as the file in which to store downloaded data.

If you don't want to specify a file, you can use `--` to separate command-line arguments from the targets:
```
instaloader.py --latest-stamps -- selenagomez
```

If you add another option after `--latest-stamps`, that also indicates to the command line parser that there is no argument for `--latest-stamps`.

**Comment by schleeb** [comment]
Duh, the command line would've helped!

That said, your guess was 100% correct, that's essentially what I was doing. I adjusted it as you said (with another command after it, which I think I actually did try previously), but it subsequently seems to want to download everything all over again, unless I also paired it with `--fast-update`. However, after running it once w/ `--fast-update`, it subsequently seems to work as expected.

For reference, the command I used this time was `.\instaloader.exe --no-video-thumbnails --no-captions --no-metadata-json --latest-stamps --no-resume [accountName]`

I think perhaps the documentation could use some clarity, but I do now understand what's needed to get it to work as expected!

Thank you!
